File: sdkwork-app-sdk-csharp/Models/GenerationHistoryVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class GenerationHistoryVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Id { get; set; }
        public string? Type { get; set; }
        public string? TaskId { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public string? Status { get; set; }
        public int? Duration { get; set; }
        public string? ResultUrl { get; set; }
        public string? ResultPreview { get; set; }
        public string? StartTime { get; set; }
        public string? EndTime { get; set; }
        public string? ModelName { get; set; }
        public int? TokenUsage { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/generation_history_vo.go
package types

// 生成历史响应
type GenerationHistoryVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id string `json:"id"`
	Type string `json:"type"`
	TaskId string `json:"taskId"`
	Title string `json:"title"`
	Description string `json:"description"`
	Status string `json:"status"`
	Duration int `json:"duration"`
	ResultUrl string `json:"resultUrl"`
	ResultPreview string `json:"resultPreview"`
	StartTime string `json:"startTime"`
	EndTime string `json:"endTime"`
	ModelName string `json:"modelName"`
	TokenUsage int `json:"tokenUsage"`
}
